packages/cli/src/utils/updaters.ts

```
@@ -0,0 +1,66 @@
+import { parse, Declaration, Rule, type AtRule } from "postcss";
+import type { CssVars } from "./registry/schema.js";
+
+export function updateCssVars(source: string, cssVars: CssVars): string {
+	if (Object.keys(cssVars).length === 0) return source;
+
+	const ast = parse(source);
+
+	// TODO: Consider adding status logs for each update. Perhaps this function could return a `string[]` of them.
+
+	// TODO: Should we error out if we fail to find/update any of the selectors?
```

I think it's better to be permissive here. Some projects won't have all selectors (like .dark) and it's fine to just skip updates we can't make rather than error out

---

eh but then again maybe failing silently isn't good either 🤔